tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -951,6 +921,10 @@ module.exports = {
     },
     filesTableHeader: {
       selector: '#files-table-header'
+    },
+    resourceDivSelector: {
```

Comment:
isn't that what we get from `getFileRowSelectorByFileName()`?

Reply:
I've removed this element and use `clickRow` now...